lib/common/app_methods.dart

```diff
@@ -36,14 +36,34 @@ class AppMethods {
   }
 
   static Future<bool> isPlayStoreEnable() async {
-    if(!Platform.isAndroid){
+    if (!Platform.isAndroid) {
       return false;
     }
     final isPlayVersion = sessionModel.isStoreVersion.value ?? false;
```

Comment:
I tried testing these changes by setting FORCE_PLAY_VERSION=true, and It looks like isPlayVersion always returns false. Are we setting storeVersion on the session model to `BuildConfig.PLAY_VERSION`?

Reply:
For me, it is working, Could you confirm that your is not a pro?